### ISSUE TITLE
Still Huge Issue With Pokestops

### ISSUE THREAD

**Issue by swiffy96** [NoxxDev/NecroBot#646]
The bot somehow stopped looting Pokestops in the v.0.9.6, even though it's next to them.. It never loots them as long as you have "CatchPokemon" set to true.. It's also not looting pokestops when walking to a Pokemon that's been sniped, even though the "SpinWhileWalking" is set to true.

This is a huge problem. I'm running out of Pokeballs all the time.

**Comment by TLBM** [comment]
Mine also ignores some PokeStops while they are just next to the gpx-track.
Others it spins "from a mile away", as soon as they are in range.

**Comment by CreativiTimothy** [comment]
Yeah. Well, for me, it does loot them when CatchPokemon is true. However, it never spins pokestops while sniping, which is a big issue

**Comment by swiffy96** [comment]
After further investigation, it seems everything works fine as long as the Human Walk Sniper system is disabled.

**Comment by TLBM** [comment]
I don't have HumanWalkSnipe or any other sniping enabled.

**Comment by sev66** [comment]
swiffy96 u mean disable this?

},
  "HumanWalkSnipeConfig": {
    "DisplayPokemonList": true,
    "MaxDistance": 1500.0,
    "MaxEstimateTime": 900.0,
    "CatchEmAllMinBalls": 50,
    "TryCatchEmAll": true,
    "CatchPokemonWhileWalking": true,
    "SpinWhileWalking": true,
    "AlwaysWalkback": false,
    "SnipingScanOffset": 0.025,
    "Enable": true,
    "WalkbackDistanceLimit": 300.0,
    "IncludeDefaultLocation": true,
    "UsePokeRadar": true,
    "UseSkiplagged": true,
    "UseSnipePokemonList": true

**Comment by swiffy96** [comment]
Yes. If you set "Enable" to false in the HumanWalkSnipeConfig, the bot works just fine.

**Comment by iwean** [comment]
I guess if you disable "TryCatchEmAll" it will spin pokestop before 50 balls run out.
